File: BAEKJOON/Python/2920.py
def check (a,b):
    for i in range(8):
        if a[i] != b[i]:
            return False
    return True


lis = list(map(int,input().split()))
temp1 = list(lis)
temp2 = list(lis)
temp1.sort()
temp2.sort()
temp2.reverse()


if check(lis,temp1):
    print("ascending")
elif check(lis,temp2):
    print("descending")
else:
    print("mixed")

File: BAEKJOON/Java/2920.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String str = String.valueOf(br.readLine());
        String[] data = str.split(" ");

        ArrayList<Integer> array = new ArrayList<>();

        for (String i : data) {
            array.add(Integer.parseInt(i));
        }

        // Check ascending
        for (int i = 0; i < 8; i++) {
            if (array.get(i) != i + 1) {
                break;
            }
            if (i == 7) {
                System.out.println("ascending");
                return;
            }
        }

        // Check descending
        for (int i = 0; i < 8; i++) {
            if (array.get(i) != 8 - i) {
                break;
            }
            if (i == 7) {
                System.out.println("descending");
                return;
            }
        }
        // Print "mixed"
        System.out.println("mixed");
    }
}
